src/main/java/string/Exer11.java

```diff
@@ -0,0 +1,40 @@
+package string;
+
+public class Exer11 {
+
+    public String subNumbers(String num1, String num2) {
+        StringBuilder result = new StringBuilder();
+        if ((num2.length() > num1.length()) || (num2.length() == num1.length() && num2.compareTo(num1) > 0)) {
+            return "-" + subNumbers(num2, num1);
+        }
+        int temp;
+        int carry = 0;
+
+        for (int i = 0; i < num1.length(); i++) {
+            temp = digitFromRight(num1, i) - digitFromRight(num2, i) + carry;
```

Comment:
Biến temp chỉ được dùng nội bộ trong vòng lặp nên khai báo trong đó thôi

```suggestion
            int temp = digitFromRight(num1, i) - digitFromRight(num2, i) + carry;
```

Reply:
fixed